libs/labelbox/src/labelbox/data/annotation_types/data/raster.py

```diff
@@ -24,7 +24,8 @@ class RasterData(BaseModel, ABC):
     uid: Optional[str] = None
     global_key: Optional[str] = None
     arr: Optional[TypedArray[Literal['uint8']]] = None
-    model_config = ConfigDict(extra="forbid", copy_on_model_validation="none")
+    
+    model_config = ConfigDict(extra="forbid")
```

Comment:
why was `copy_on_model_validation` removed

Reply:
![image](https://github.com/user-attachments/assets/b729353b-5151-4cf0-92ce-a8fb5fb81a88)
Pydantic V2 removed it :( with no alternative

Reply:
![image](https://github.com/user-attachments/assets/ac6d49ab-cc35-462c-a033-bb8a92e81cdc)
This is what it used to do. We don't have a way to add this behavior anymore but Im not sure why the original author added this setting it is the only place that includes this in the config settings